src/tests/electron/tests/automated-checks-view.test.ts

```diff
@@ -1,15 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import * as path from 'path';
+import { UnifiedFeatureFlags } from 'electron/common/unified-feature-flags';
 import { createApplication } from 'tests/electron/common/create-application';
 import { scanForAccessibilityIssuesInAllModes } from 'tests/electron/common/scan-for-accessibility-issues';
 import { AppController } from 'tests/electron/common/view-controllers/app-controller';
 import { CardsViewController } from 'tests/electron/common/view-controllers/cards-view-controller';
+import { ResultsViewController } from 'tests/electron/common/view-controllers/results-view-controller';
 import { commonAdbConfigs, setupMockAdb } from 'tests/miscellaneous/mock-adb/setup-mock-adb';
 
 describe('AutomatedChecksView', () => {
     let app: AppController;
     let cardsView: CardsViewController;
+    let resultsView: ResultsViewController;
```

Comment:
Can this be scoped inside `openResultsAndCardsViews`?

Reply:
Oh, right. Good catch! Updating